lambdas/account-data-deleter-batch-delete/src/config.ts

```diff
@@ -1,8 +1,8 @@
 const shortName = 'ADD';
 
 const awsEnvironments = ['production', 'development'];
-let localAwsEndpoint = undefined;
-if (!awsEnvironments.includes(process.env.NODE_ENV)) {
+let localAwsEndpoint: string | undefined = undefined;
```

Comment:
don't need to change this, but I'd prefer a ternary statement vs. initialization and conditional assignment for our code generally

Reply:
@kschelonka `const localAwsEndpoint =
  process.env.NODE_ENV && !awsEnvironments.includes(process.env.NODE_ENV)
    ? process.env.AWS_ENDPOINT || 'http://localhost:4566'
    : undefined;`